### DISCUSSION TITLE
Is `atom.io` offline? `discuss.atom.io` has invalid certificate?

### DISCUSSION THREAD

**Discussion by ioquatix** [atom/atom#22746] [General]
Visiting `blog.atom.io` has a link to `https://discuss.atom.io/` which presents a certificate warning:

<img width="720" alt="image" src="https://user-images.githubusercontent.com/30030/126085863-b1327e1e-896c-4ac8-8755-50244023ae93.png">

`atom.io` seems offline?

<img width="737" alt="image" src="https://user-images.githubusercontent.com/30030/126085878-d0e611fe-59b6-4138-942e-b7e33c64eaa9.png">

What's going on?

**Comment by TechStudent10** [comment]
`atom.io` works for me.

**Reply by ioquatix** [reply]
Yes, it is working now. It was offline for a while.

**Comment by maleksware** [comment]
Atom discussions have moved here, to GitHub. [discuss.atom.io](discuss.atom.io) was closed because of this.

**Reply by ioquatix** [reply]
The `atom.io` website still links to `discuss.atom.io`:

<img width="121" alt="image" src="https://user-images.githubusercontent.com/30030/126607794-57eda7ec-eea2-4775-aaea-9c84e1a56415.png">

If you click that link, you get a security warning.

**Reply by ghost** [reply]
So much Atom guidance and Q&A information was lost. These pages should have been archived..

**Reply by maleksware** [reply]
> 
> 
> So much Atom guidance and Q&A information was lost. These pages should have been archived..

As Atom is a GitHub product, they could probably find a way to move everything. But I totally agree that so much info can be lost forever :(

**Reply by ghost** [reply]
I mean, I've tried implementing a forum migration before and it's generally an awful time for the developer. Unless the goal is extracting a formatted static backup. The key issue is that there isn't a common, middle-man standard or protocol for forum data that migration processes could use.

**Comment by maleksware** [comment]
[atom.io](atom.io) is OK and working. Discussion subdomain no longer works, it has moved here.

**Reply by ioquatix** [reply]
Maybe they should add a redirect.

**Reply by ferdnyc** [reply]
@ioquatix They did, but they did it by redirecting the DNS record — that's the problem. `discuss.atom.io` is now a CNAME for `redirect.github.com`, which returns a security certificate for `CN=*.github.com`. That fails to match the target hostname.

**Reply by 2brownc** [reply]
I guess that is the reason why `http://discuss.atom.io/` redirects without any problem but not `https://discuss.atom.io/`.
